src/Legacy/Framework/Initializer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+
+namespace Contao\CoreBundle\Legacy\Framework;
```

Comment:
I don't like the naming. Especially we want to omit `Legacy` because it's not. It is the "old way" but it's not legacy because there is no alternative to it yet.

I think `Contao\CoreBundle\Framework` is correct because we can then put other "stand-alone" services there as well. And I think the class should be named `FrameworkInitalizer` so it's better readable. I talked to @aschempp about the naming already and this makes sense to us both.